taf/testlib/virtual_env.py

```diff
@@ -271,12 +273,17 @@ def ensure_public_access(self, try_reuse=False, networks_client=None, routers_cl
             tenant_id = self.tenant_id
 
         _net_cfg = self.config.network
-        _mgmt_ip_cidr = self.settings.get('mgmt_ip_cidr')
-        assert _mgmt_ip_cidr
-        net_ip = netaddr.IPNetwork(_mgmt_ip_cidr)
+        _external_net_ip_cidr = self.settings.get('external_net_ip_cidr')
+        assert _external_net_ip_cidr
+        net_ip = netaddr.IPNetwork(_external_net_ip_cidr)
+
+        _external_net_pool_start = self.settings.get('external_net_pool_start')
+        assert _external_net_pool_start
+        _external_net_pool_end = self.settings.get('external_net_pool_end')
+        assert _external_net_pool_end
```

Comment:
Yes, please add any constants instead of using magic numbers.  The value 100 seemed to be the previous minimum, see line 304 of the previous revision and line 305 for the value 254.  If we have a reason to open up this range, then make those values instead of mine.